notary.go

```diff
@@ -6,6 +6,9 @@ package notary
 // decide how many chances to give to a human, for example.
 type PassRetriever func(keyName, alias string, createNew bool, attempts int) (passphrase string, giveup bool, err error)
 
+// PassValidator is a callback function to validate a passphrase. It returns an error if validation fails.
+type PassValidator func(string) error
```

Comment:
Still wondering if this should be in the `notary` package, or if it should be moved to `passphrase` (so it would become `passphrase.Validator`)